test/ctr.bats

```diff
@@ -1251,6 +1251,11 @@ function teardown() {
 	[ "$status" -eq 0 ]
 	[[ "$output" =~ "209715200" ]]
 
+	run crictl exec --sync "$ctr_id" sh -c "cat /sys/fs/cgroup/memory/memory.memsw.limit_in_bytes || cat /sys/fs/cgroup/memory.swap.max"
```

Comment:
the swap limits work differently on cgroup v2.

in cgroup v1 the memory swap limit accounts also for the memory max, so `memory.memsw.limit_in_bytes` is really `memory limit + swap`, while on cgroup v2 it is just the net `swap` limit.

If you try on cgroup v1 to set `memory.memsw.limit_in_bytes` lower than `memory.mem.limit_in_bytes` the kernel will complain.

`crun` maintains the same semantic of the cgroup v1 OCI configuration also with cgroup v2, so the value that is set for `SetLinuxResourcesMemorySwap()` will be converted to `SWAP_LIMIT - MEMORY_LIMIT`:

on cgroup v1:
```
# podman run --rm -ti --memory 100M --memory-swap=100M fedora cat /sys/fs/cgroup/memory/memory.memsw.limit_in_bytes 
104857600
```

on cgroup v2:
```
# podman run --rm -ti --memory 100M --memory-swap 100M alpine cat /sys/fs/cgroup/memory.swap.max
0
```

both of these containers cannot use any swap memory.

Reply:
Is this the same behavior we get from runc?  If so, I'm fine with changing the logic here to set the swap value to double the memory value.

Reply:
it will eventually follow the same behavior otherwise it is a breaking change for the cgroup v1 configuration

Reply:
Right, so the code's doing the right thing (effectively disabling swap), but the test needs to be fixed.  Got it.